server/main.lua

```diff
@@ -18,41 +18,42 @@ end
 local function giveStarterItems()
     local Player = QBCore.Functions.GetPlayer(source)
     if not Player then return end
+    local citizenid, firstname, lastname, gender, birthdate, nationality = Player.PlayerData.citizenid, Player.PlayerData.charinfo.firstname, Player.PlayerData.charinfo.lastname, genderString(Player.PlayerData.charinfo.gender), Player.PlayerData.charinfo.birthdate, Player.PlayerData.charinfo.nationality
```

Comment:
nit; prefer separate lines for declaration of variables to make it easier to read